src/script/interpreter.cpp

```diff
@@ -1860,6 +1860,15 @@ bool GenericTransactionSignatureChecker<T>::CheckSequence(const CScriptNum& nSeq
     return true;
 }
 
+template <class T>
+bool GenericTransactionSignatureChecker<T>::CheckDefaultCheckTemplateVerifyHash(const std::vector<unsigned char>& hash) const
```

Comment:
any reason to not use a `Span` here?

Reply:
i think it didn't exist when i coded this haha. a span would be fine if i touch it.

Reply:
doing this one -- I'm not confident I like it so I'll leave it as a patch until you ACK it